boards/arm/b_u585i_iot02a/b_u585i_iot02a.dts

```diff
@@ -24,3 +24,38 @@
 		sw0 = &user_button;
 	};
 };
+
+&flash0 {
+	/*
+	 * For more information, see:
+	 * https://docs.zephyrproject.org/latest/reference/devicetree/index.html#fixed-flash-partitions
+	 */
```

Comment:
Please remove this

Reply:
cf https://github.com/zephyrproject-rtos/zephyr/pull/43882

Reply:
removed